packages/@aws-cdk/aws-cloudfront/test/test.basic.ts

```diff
@@ -572,6 +572,7 @@ export = {
             "Aliases": [],
```

Comment:
I'm also happy to break out different tests for defaults, as opposed to re-using the "basic usage" tests. I didn't quite know what the preference would be.